index.js

```diff
@@ -28,6 +28,14 @@ var ERROR_TYPES = {
     INVALID_ARGUMENTS: 'invalid_arguments',
 };
 
+if (!String.prototype.startsWith) {
```

Comment:
I'm sure this is my poor JS knowledge - but why do we need to check if String has a startsWith method? 

I guess "polyfill for <= 0.12" is saying for older versions of JS? Do we need to support them?